src/styles/blocks/card.scss

```diff
@@ -0,0 +1,82 @@
+.card {
+  width: $card-dimension;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  color: $description-color-text;
+
+  &__image {
+    margin-bottom: 40px;
+    align-self: center;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: $ff-500;
```

Comment:
What is ff? Let's make more clear names for variables

Reply:
font-fat)